pkg/api/core/v1/client/http.go

```diff
@@ -154,7 +154,12 @@ func (c *Client) do(endpoint, method, requestBody string) ([]byte, error) {
 	// TODO why is != 200 an error? there are valid codes in the 2xx, 3xx range
 	if response.StatusCode != http.StatusOK {
 		err := formatError(bodyBytes, response)
+
+		if respLog.V(15).Enabled() {
+			respLog = respLog.WithValues("body", string(bodyBytes))
```

Comment:
can we provide more logging, maybe some http headers, that allow us to identify the server? In case it's a loadbalancer or DNS issue

Reply:
Yes, I've actually haven't added it because it was already there, but I've used the same logLevel

https://github.com/epinio/epinio/blob/bbe4a58416d43b263e83286dc0f203d37e590d31/pkg/api/core/v1/client/http.go#L247-L256

Reply:
So we didn't see it in the report, because the log level was not high enough. That's unfortunate.

I wonder how large the body can be for a request that's not of status code 200. Anyhow, 15 feels like a high log level.